airbyte-cdk/python/airbyte_cdk/sources/declarative/requesters/error_handlers/backoff_strategy.py

```diff
@@ -9,6 +9,10 @@
 
 
 class BackoffStrategy:
+    """
+    Backoff strategy defining how long to backoff before retrying a request that resulted in an error.
```

Comment:
Rewrote a little bit to avoid reusing the word backoff when describing the backoff.
`Backoff strategy defining how long to wait before retrying a request that resulted in an error.`

Reply:
@brianjlai 
just as fyi because it's a cool hidden feature, you can write a [suggestion block](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request) with "```suggestion" 

```suggestion
    Backoff strategy defining how long to wait before retrying a request that resulted in an error.
```